CHANGELOG.md

```diff
@@ -13,8 +13,8 @@
 
 
 ### Features
-
-
+- Fixes an instance of two runways called "34R" in Shanghai Pudong [#149](https://github.com/openscope/openscope/issues/149)
```

Comment:
tried git pull origin develop then pushed the changes however this diff did not go away